backend/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:10-alpine
+
+# Set environment variables.
+ENV NODE_ENV production
+ENV DEBUG app:info,app:warn,app:error
+
+# Install dependencies.
+RUN apk update && \
+  apk add --no-cache \
+    --virtual build-dependencies \
+      build-base \
+      gcc \
+      wget \
+      git
```

Comment:
I tend to recommend ordering these by alphabet.

Reply:
Good idea!

Reply:
Done